ifrs17/Import/Importers.ipynb

```diff
@@ -582,6 +582,79 @@
             "execution_count": 0,
             "outputs": []
         },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Yield Curves "
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "Import.DefineFormat(ImportFormats.YieldCurve, async (options, dataSet) => {",
+                "\n    Activity.Start();",
+                "\n    // Parsing Main",
+                "\n    var mainTab = dataSet.Tables[Main];",
```

Comment:
Extract common part of parsing of args in one method

Reply:
done